c/apps/lf-workspace/hello/src/Makefile

```diff
@@ -0,0 +1,38 @@
+LF_PROJECT_ROOT := "/home/ehsan/t-crest/patmos/c/apps/lf-workspace/hello/"
+LF_MAIN_TARGET := HelloWorld
```

Comment:
Does this hardcode the program to be naked `HelloWorld`. Shouldnt you be able to use another name?